css/_variables.scss

```diff
@@ -212,7 +212,7 @@ $welcomePageTabContentDisplay: inherit;
 $welcomePageTabButtonsDisplay: flex;
 $welcomePageTabDisplay: block;
 
-$welcomePageButtonWidth: 51px;
+$welcomePageButtonWidth: 75px;
```

Comment:
This is a magical number and would needs a better fix for a longer string, isn't it?

Reply:
Setting a width with a fixed number should be avoided anyway in terms of localization IMO.